File: Trip-Planner/src/app/classes/Land.ts
import { Continent } from "./Continent";
import { Plaats } from "./Plaats";

export class Land {
    landId: number;
    naam:string;
    continent: Continent;
    plaatsen: Plaats[] = [];
    
    constructor(landId: number, naam:string, continent: Continent,) {
        this.landId = landId;
        this.naam = naam;
        this.continent = continent;
    }
}

File: TripPlannerAPI/TripPlannerDAL/Entity/Land.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TripPlannerDAL.Entity
{
    public class Land
    {
        public int landId { get; set; }
        public string naam { get; set; }
        public int? continentId { get; set; }
        public Continent? continent { get; set; }
        public ICollection<Plaats>? plaatsen { get; set; }
    }
}
